packages/above/above.install

```diff
@@ -0,0 +1,17 @@
+post_install() {
+  set -e
+  cd /usr/share/above
+  python -m venv venv
+  source venv/bin/activate &&
+    pip install --isolated --root="/usr/share/above" --prefix='venv' \
```

Comment:
the dependencies look simple, why use a virtual env? https://github.com/casterbyte/Above/blob/4789d6eed0dce7aaf6103852fe7915c6e1d56a5d/setup.py#L16-L18